### PR TITLE
setting up Opacity of the Userbutton

### PR DESCRIPTION
**Pull request by blackcrack**
Opacity by normal should be less and by hover should by 1 
because then it is active .. 
in other hand should normally 
normal = .5
hover = .8
and pressed/active = 1
how ever, i hope i have fix it now..
best regards
Blacky

### CODE REVIEW

**Comment by rullzer** [comment]
@nextcloud/designers

**Comment by skjnldsv** [comment]
Ahah, this is for @jancborchardt this :)

**Comment by jancborchardt** [comment]
Yes it is the inverse of the other app icons, which are full opacity _on hover_.

But we actually had that at some point, and it looks very strange when the user’s profile image is not fully opaque / visible by default. Especially depending on the color of the header bar (or if even an image is set) it can look very off.

This is why we decided to have the user’s profile image in the header at full opacity by default. You can also check out other web services, and basically everyone shows the image at full opacity.

If the "transparency on hover" is not good, we could also do other things to highlight. For example put a white outline around the image on hover/active.

**Review by jancborchardt — CHANGES_REQUESTED** [review]
See my comment

**Comment by blackcrack** [comment]
sorry if i speak there in german, but it's more uncomplicated.. for less misunderstood 
@jancborchardt 
ja klar, aber wenn vornerein schon auf .53 eingestellt ist (was soll eigendlich die krumme zahl(3) bewirkt kaum was)
also hier ist es so, daß der Userbutton erstmal standart mit .53 und nicht mit voller deckkraft eingestellt ist, was ich eben grad ändern wllte, weil standartmässig 0.53 übel aussieht,
aber wenn man dann draufdrückt also hover und active macht, dann wird das bild ja nichtig (also nichtmehr wiichtig und dann tritt das Menu hervor.. genau das wollt ich ja machen .. und nich andersrum..

also, sind wir konform.. schau dir bitte nochmal den code an und die veränderung die ich gemacht habe..
ich hätte sogar gern noch hover und active eingebaut.

liebe Grüße
Blacky

**Comment by MorrisJobke** [comment]
> ich würd sogar mit hover und activ arbeiten, da auf dem header ja generell die icons mit opacity
> eingestellt sind, wenn man dann hover sollte das Userbild dann voller angezeigt werden
> und danach nach active mit 1, somit währe dann die conformität mit den anderen Such und Chat sowie anouceing buttons conform ist.

I don't understand this one :/ Could you rephrase this?




> Mal was anderes da wir grad schon im design sind,

This is not how issue trackers work - please stick to one topic per issue. Otherwise it gets super confusing for everyone.

But let me answer your question regarding themes: we are on the way to phase out the "themes" and only use the "theming app", because this allows us to have a unified look and feel on the one hand and give the users the ability to slightly (!) tweak the UI to their needs (basically the colors, slogans and logos). Changing the whole look and feel is nothing that is really wanted to be done, because it will break in 99,999% of the cases for some UI elements. Thus putting more effort into "themes" is not on our roadmap and will not be in the foreseeable future.

I hope this makes it clear where we are heading with this project.

**Comment by blackcrack** [comment]
this project goes more and more "professional" and no more really for the private-community.. you want make money , right ? And there it is no Space anymore for Private and modding Designer ..
okey, so far have i understand this .. "and it's been called, we want to support the community and work with them...." wohw.. oookey... now be i am a bit disappointed .. and now I am no longer deceived ..

thank you Morris for clear words ..

**Comment by MorrisJobke** [comment]
> this project goes more and more "professional" and no more really for the private-community.. you want make money , right ? And there it is no Space anymore for Private and modding Designer ..
> okey, so far have i understand this .. "and it's been called, we want to support the community and work with them...." wohw.. oookey... now be i am a bit disappointed .. and now I am no longer deceived .

I was not about to disappoint you. I just want to make a point on what we focus. And this does not mean we do not care about our community, but the majority of the community is not about "modding" their instance. They have a need for a file sharing solution plus a lot of features that are provided by apps. Our major effort is to bring a stable platform for those solutions. Thus we need to focus on given aspects and we will never be able to satisfy everybody by 100% but to make the best possible solution that is out there, that does the stuff we want to accomplish (handling your files in an hassle free UI for example). Thus we also need to cut some "freedoms" like the one to tweak any possible part of the UI to make it possible to develop apps for Nextcloud that work across multiple versions of Nextcloud and do not break all the time.

That is our goal and we work based on feedback from the community and from customers (which are very often also part of the community and were only customers because of this community).

On the other hand we do not receive many feedback that demands a customization without limits. Many many people just came with positive feedback about the theming app because it made so many peoples live easier and Nextcloud more stable.

I would like you to also see it from the maintainers perspective: at some point we need to say "no" to some features, because we have limited time and need to invest this limited time into the stuff that matters for most of the people out there. Thus we spend less effort on the themes section but more effort on the robustness of the underlying DB, file storage integration or the general UI/UX.

I hope that makes it a bit more clear to you, why we do the work we do.

**Comment by MorrisJobke** [comment]
And if you really want to tweak the UI up to the limits of CSS: You could of course adjust the source code whenever you want and don't need to think about the CSS that is not needed. You could delete that part and replace it by the code you want to be there. After all we are an open source project that gives you this freedom at any time. And we can not and don't want to limit that at any time.

**Comment by blackcrack** [comment]
Dear Morris,

you have handle me ever correct and this don't i forget that ":smile_thumpsup: "
i be not disappoint about you.. you total okey and a nice Guy !

but i think, there was forgot to must exist earth and space for for growing .. up ..
it has never give a Theme-upload possibility, it has never give a place for speak about themes.. 
and nothing about was really thinking for give space for the Designer, no wonder to was there never really a grown up of the Theme-area and therewith so very less feedback of the private Community-space
because the Designer was run away in the start of Nextcloud , also because it has no give a space ..

And broken of the theme can you not bring on the table, because the standard theme must ever be adapted, maintained and upgraded  .. and say not to me , i lie :) \*vbs* 

all what the theme designer as other themes building up, have they are to maintaining and upgrading
by self . Works no more, fly it out of the collecting or stay on the Version for what it was build.
My hobby, i maintaining my Nextcloudtheme and put it on my own space for download.
Because it is give no space on or by the Nextcloud... Community , so have i am hosting by self
on my blackysgate.de/files .. \<retoric>So, now comes you ..\</retoric>

how the hack makes PHPBB .. only as -one- example 
or other webaplications where it is possible to theming and comes out from the community .. ?

so..

best regards
Blacky

**Comment by MariusBluem** [comment]
We have discussed the possibility of adding a theme store in the past already and we came up with the small use-case of such a feature. 90% of administrators who wants to create a theme are interested in bringing their corporate identity to Nextcloud by adding their own logo and change the color scheme (which is covered with our theming-app) ... not more and not less.

The old `/themes/` in the folders are still supported and you are of course allowed to use this. Same for the `theming_customcss`-app you may have heard from: https://apps.nextcloud.com/apps/theming_customcss

Organizations do not want / need to share their customizations with the public since they are to special and only for internal use (own logo, ...). For home-use the default Nextcloud-layout (which can be adjusted with an own color and logo if wanted) is mostly enough. If you want more - do it by using one of those ways I have described above.

I will close this request now, since the reasons @jancborchardt has described above.
